x-pack/legacy/plugins/canvas/__tests__/fixtures/function_specs.ts

```diff
@@ -6,8 +6,9 @@
 
 // @ts-ignore Untyped Library
 import { Fn } from '@kbn/interpreter/common';
+import { uniq } from 'lodash';
 import { functions as browserFns } from '../../canvas_plugin_src/functions/browser';
 import { functions as commonFns } from '../../canvas_plugin_src/functions/common';
 import { functions as serverFns } from '../../canvas_plugin_src/functions/server';
```

Comment:
Not really in the scope of this PR, but should we be importing the functions under `public/functions` as well? They're the only ones that live outside of the `canvas_plugin_src` dir.